resources/darwin/bin/nerdctl

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+scriptdir="${BASH_SOURCE[0]}"
+[ -L "${scriptdir}" ] && scriptname="$(readlink "${scriptdir}")"
+scriptdir="$(cd "$(dirname "${scriptname}")" && pwd)"
+
+if ! LIMA_HOME="$HOME/Library/State/rancher-desktop/lima" ${scriptdir}/../lima/bin/limactl ls --json | grep '"name":"rancher-desktop"' | grep -q '"status":"Running"'; then 
```

Comment:
```suggestion
if ! LIMA_HOME="$HOME/Library/State/rancher-desktop/lima" "${scriptdir}/../lima/bin/limactl" ls --json | grep '"name":"rancher-desktop"' | grep -q '"status":"Running"'; then 
```

same thing in line 10

Reply:
Good catch. Got this and the other instance this could happen.